include/constants/event_objects.h

```diff
@@ -271,7 +271,7 @@
 #define OBJ_EVENT_GFX_VAR_F  (OBJ_EVENT_GFX_VARS + 0xF)
 
 #define OBJ_EVENT_GFX_MON_BASE  0x200 // 512
-#define OBJ_EVENT_GFX_SPECIES_BITS 11
+#define OBJ_EVENT_GFX_SPECIES_BITS 12
```

Comment:
```suggestion
#define OBJ_EVENT_GFX_SPECIES_BITS 12 // This will need to be updated when NUM_SPECIES is > XXX
```

Reply:
Is there a way to generate this number using a fancy macro that takes uses `NUM_SPECIES`? Even something ugly like [`MAX_MONEY`](https://github.com/PokemonSanFran/pokeemerald-expansion/blob/18aff2db3bf310d703cd548c167f9aa447a319ca/include/money.h#L6) would be nice imho.

Reply:
MGriffins math is [here](https://github.com/rh-hideout/pokeemerald-expansion/issues/5105#issuecomment-2271291491)
So we should be good until ~3.5k? Maybe?